strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/util/PypiWheelArtifactCoordinatesUtils.java

```diff
@@ -0,0 +1,57 @@
+package org.carlspring.strongbox.util;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiWheelArtifactCoordinates;
+
+/**
+ * Class to handle parsing of PyPi filename string
+ * 
+ * @author alecg956
+ */
+public class PypiWheelArtifactCoordinatesUtils
+{
+    /**
+     * If optional build parameter is not found in the filename the empty string is specified for build_tag
+     * in the construction of a PypiWheelArtifactCoordinates object
+     * 
+     * Format of Wheel: {distribution}-{version}(-{build tag})?-{python tag}-{abi tag}-{platform tag}.whl.
+     * 
+     * @param path: The filename of the PyPi Wheel artifact
+     * @return Returns a PypiWheelArtifactCoordinate object with all coordinates in the filename set
+     */
+    public static PypiWheelArtifactCoordinates parse(String path)
+    {
+        String[] splitArray = path.split("-");
+
+        // check for invalid file format
+        if (splitArray.length != 5 && splitArray.length != 6)
+        {
+            throw new IllegalArgumentException("Invalid Wheel filename specified");
+        }
+
+        String distribution = splitArray[0];
+        String version = splitArray[1];
+        String build = "";
+        String lang_impl_version = "";
```

Comment:
A few of things:
* In Java, we typically follow the camel hump convention (`fooBar`, instead of `foo_bar`).
* The convention in our project for variables is more along the lines of:
  * `InputStream is = ...`, or `InputStream inputStream`
  * Field names use the full variable name
* Could you please rename this variable to `languageImplementationVersion`?